django_cloud_deploy/tests/lib/test_base.py

```diff
@@ -87,8 +87,7 @@ def setUp(self):
         self.project_dir = tempfile.mkdtemp()
         image_name = utils.get_resource_name(resource_type='image')
         self.image_tag = '/'.join(['gcr.io', self.project_id, image_name])
-        self.instance_name = utils.get_resource_name(
-            resource_type='sql-instance')
+        self.instance_name = utils.get_resource_name(resource_type='instance')
```

Comment:
When parsing resource creation time, I used string.split('-'). The change here is to avoid unexpected results.